Cargo.lock

```diff
@@ -1486,7 +1486,7 @@ dependencies = [
 [[package]]
 name = "cs_derive"
 version = "0.1.0"
-source = "git+https://github.com/matter-labs/era-sync_vm.git?tag=v1.3.3-rc0#23beb8f268427508aefc916cff0d42a36b877586"
+source = "git+https://github.com/matter-labs/era-sync_vm.git?branch=v1.3.3#e819d15b107a06a746299f98bbd9802e26eeb348"
```

Comment:
nit/unrelated: Should we be using `tag` instead of `branch`? Unsure where in the chain this is being set to branch

I think this is coming from here: https://github.com/matter-labs/zksync-era/blob/a43bc402a204ac92fb74ad348b68f61e24d7a84a/core/lib/utils/Cargo.toml#L14

Reply:
we moved back to tracking 'branches' - and will update the versions more often.